models/dataset_test.go

```diff
@@ -100,8 +101,8 @@ func TestCreateVersion(t *testing.T) {
 		Convey("when the version has all fields", func() {
 			b, err := json.Marshal(associatedVersion)
 			if err != nil {
-				log.ErrorC("Failed to marshal test data into bytes", err, nil)
-				os.Exit(1)
+				t.Logf("failed to marshal test data into bytes, error: %v", err)
```

Comment:
I think it would be more concise to use `So(err, ShouldBeNil)`

Reply:
See response on previous comment